packages/apidom-ns-openapi-3-1/test/refractor/plugins/normalize-parameter-examples/storage-field.ts

```diff
@@ -0,0 +1,70 @@
+import { assert } from 'chai';
+import dedent from 'dedent';
+import { toValue } from '@swagger-api/apidom-core';
+import { parse } from '@swagger-api/apidom-parser-adapter-yaml-1-2';
+
+import { OpenApi3_1Element, refractorPluginNormalizeParameterExamples } from '../../../../src';
+
+describe('refractor', function () {
+  context('plugins', function () {
+    context('normalize-parameter-examples', function () {
+      specify('should use sub-field to store normalized scopes', async function () {
+        const yamlDefinition = dedent`
+            openapi: 3.1.0
+            paths:
+              /:
+                get:
+                  parameters:
+                      - in: query
+                        name: idempotent
+                        schema:
+                          type: number
+                          example: 1
+                        examples:
+                          example1:
+                          value: 2
+  `;
+        const apiDOM = await parse(yamlDefinition);
+        const openApiElement = OpenApi3_1Element.refract(apiDOM.result, {
+          plugins: [refractorPluginNormalizeParameterExamples()],
+        }) as OpenApi3_1Element;
+
+        assert.deepEqual(toValue(openApiElement.get('x-normalized')), {
+          'parameter-examples': ['/paths/~1/get/parameters/0'],
+        });
+      });
+
+      context('given custom storage field', function () {
+        specify('should use custom sub-field to store normalized scopes', async function () {
```

Comment:
```suggestion
        specify('should use custom storage to store normalized scopes', async function () {
```

Please fix in my original plugin tests as well